### PR TITLE
docs(links): add version01

### PR DESCRIPTION
**Pull request by kibos11**
There are many things that need a fix but the wiki page is still under work I think.

### CODE REVIEW

File: src/systems/ec/specs/components/link/docs/usage.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+  * more link
+  * expandable link
+  * social media link
+
+## When to use this component
+
+### in-line link
+
+When links are inserted within paragraph text blocks
+Style blue, underlined (displayed in white colour if the background colour is blue)
```
Comment (yhuard) [inline src/systems/ec/specs/components/link/docs/usage.md]: Style specs should go into the "style" tab, not "usage"

File: src/systems/ec/specs/components/link/docs/usage.md [hunk 2]
```diff
@@ -1 +1,53 @@
...
+
+Used on the Expandable component
+
+#### social media link
+
+Used on the Social media - Follow. and Social media - Share. components
+
+## When not to use 
+
+For actions or important links to other pages -  use buttons instead
```
Comment (yhuard) [inline src/systems/ec/specs/components/link/docs/usage.md]: Please do not confuse links with buttons. Links are used for navigation. Buttons are used for actions (e.g. submitting/resetting a form).

- http://www.webaxe.org/proper-use-buttons-links/
- https://medium.com/simple-human/but-sometimes-links-look-like-buttons-and-buttons-look-like-links-9b371c57b3d2

**Review by yhuard — CHANGES_REQUESTED** [review]

File: src/systems/ec/specs/components/link/docs/usage.md (hunk 2 above)
Comment (degliwe) [reply src/systems/ec/specs/components/link/docs/usage.md]: https://a11y-101.com/design/button-vs-link

Comment (kibos11) [reply src/systems/ec/specs/components/link/docs/usage.md]: I will make sure the relevant person to check it and updated. I just copy paste the content that I have in the wiki. Consider it as a draft.

**Pull request closed without merging**